packages/mdc-slider/index.js

```diff
@@ -81,8 +99,8 @@ class MDCSlider extends MDCComponent {
         hasClass: (className) => this.root_.classList.contains(className),
         addClass: (className) => this.root_.classList.add(className),
         removeClass: (className) => this.root_.classList.remove(className),
-        getAttribute: (name) => this.root_.getAttribute(name),
-        setAttribute: (name, value) => this.root_.setAttribute(name, value),
+        getAttribute: (name) => this.thumb_.getAttribute(name),
```

Comment:
You are right about getAttribute, its not being used in the foundation it would be used for getting disabled state later but i will remove. Set is being used to update all the min max and value but it is the thumb attributes so i will update this.